### DISCUSSION TITLE
May I ask if it is possible to add a spot color channel to cmyk's tif file

### DISCUSSION THREAD

**Discussion by 291598912** [python-pillow/Pillow#7154] [Q&A] [answered]
May I ask if it is possible to add a spot color channel to cmyk's tif file

**Comment by radarhere** [comment, marked as answer]
Spot color channels have been mentioned before, and the conclusion was that they aren't supported.

https://github.com/python-pillow/Pillow/issues/3110#issuecomment-384254127
> You're essentially talking about spot colors at that point... there's no support for more than 4 usable channels in pillow.

#1888 is an open issue partly about supporting more channels.
